test/integration/connect/envoy/case-cfg-resolver-cluster-peering-failover/primary/verify.bats

```diff
@@ -35,7 +35,7 @@ load helpers
 }
 
 @test "peer the two clusters together" {
-  create_peering primary alpha
+  retry_default create_peering primary alpha
```

Comment:
If only half of `create_peering` completes, will it be re-entrant and pick up where it left off or fail? Mostly I'm wondering how effective the retry would be.

Reply:
if only half of the create_peering, the retry will still redo the peering steps.

https://github.com/hashicorp/consul/blob/1c5d54cb29e6940009285cda8e575971bfbe9eec/test/integration/connect/envoy/helpers.bash#L925-L939

It seems the retry will be only useful for slow consul bootstrap, like initializing the CA.